src/stdlib_stats_distribution_normal.fypp

```diff
@@ -0,0 +1,332 @@
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+module stdlib_stats_distribution_normal
+    use stdlib_kinds, only : sp, dp, xdp, qp, int32
+    use stdlib_error, only : error_stop
+    use stdlib_random, only : dist_rand
+    use stdlib_stats_distribution_uniform, only : uni=>rvs_uniform
+
+    implicit none
+    private
+
+    real(dp), parameter  :: HALF = 0.5_dp, ONE = 1.0_dp, TWO = 2.0_dp
```

Comment:
Here I suggest you add a comment `! These variables are only used in the random number generation, which is fundamentally double precision`.